packages/react-error-overlay/src/index.js

```diff
@@ -18,22 +18,36 @@ import { iframeStyle, overlayStyle } from './styles';
 import { applyStyles } from './utils/dom/css';
 
 import type { ErrorRecord } from './listenToRuntimeErrors';
+import type { ErrorLocation } from './utils/parseCompileError';
 
 type RuntimeReportingOptions = {|
   onError: () => void,
-  launchEditorEndpoint: string,
   filename?: string,
 |};
 
+type OpenInEditorListener = (errorLoc: ErrorLocation) => void;
+
 let iframe: null | HTMLIFrameElement = null;
 let isLoadingIframe: boolean = false;
 
 let renderedElement: null | Element<any> = null;
+
+let openInEditorListener: null | OpenInEditorListener = null;
 let currentBuildError: null | string = null;
 let currentRuntimeErrorRecords: Array<ErrorRecord> = [];
 let currentRuntimeErrorOptions: null | RuntimeReportingOptions = null;
 let stopListeningToRuntimeErrors: null | (() => void) = null;
 
+export function listenToOpenInEditor(listener: OpenInEditorListener) {
+  openInEditorListener = listener;
+}
+
+function openInEditor(errorLoc: ErrorLocation) {
+  if (typeof openInEditorListener === 'function') {
```

Comment:
This is the indirection I'm referencing. Alternatively, we could add a warning, e.g. "You attempted to open a file in your editor but no handler was set".